src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -536,11 +536,11 @@
   <data name="IgnoreConditionOnValueTypeInvalid" xml:space="preserve">
     <value>The ignore condition 'JsonIgnoreCondition.WhenWritingNull' is not valid on value-type member '{0}' on type '{1}'. Consider using 'JsonIgnoreCondition.WhenWritingDefault'.</value>
   </data>
-  <data name="NumberHandlingConverterMustBeBuiltIn" xml:space="preserve">
-    <value>'JsonNumberHandlingAttribute' cannot be placed on a property, field, or type that is handled by a custom converter. See usage(s) of converter '{0}' on type '{1}'.</value>
+  <data name="NumberHandlingOnTypeInvalid" xml:space="preserve">
+    <value>Number handling is only applicable to number types or collections of numbers. When applied to a property or field, the member cannot be handled by a custom converter. If the member is a collection of numbers, the number elements cannot be handled by a custom converter. See type '{0}'.</value>
```

Comment:
> the number elements cannot be handled by a custom converter

Isn't one of the recourse that the user could take is remove the NumberHandling attribute or change `JsonNumberHandling` option value? When they hit this, depending on what they want to do, they may not want to remove the custom converter.

Consider rewording to make it clear what different actions the user can take to fix this issue.

Reply:
Perhaps something like:
`
Number handling is not compatible with custom converters. Either change the JsonSerializerOptions to not use number handling, remove the NumberHandlingAttribute from the member, or remove usage of the custom converter. See type '{0}'.`

Reply:
Maybe `custom converters` -> `user-defined converters`? But the rest reads much better.

Reply:
Thanks - I'm inclined to remove the restrictions on using custom converters. The exception message won't contain info about converters.